package-lock.json

```diff
@@ -2305,9 +2305,9 @@
       }
     },
     "@react-native-community/netinfo": {
-      "version": "4.6.1",
-      "resolved": "https://registry.npmjs.org/@react-native-community/netinfo/-/netinfo-4.6.1.tgz",
-      "integrity": "sha512-RIcrNzVnkes6/d5jFprce8i6ruUO9+/FVZZgejlu/TSnysyFXHNJKgcIqxbKx+7XfQxfjrCGeCvsfYxrFrU0zw=="
+      "version": "4.7.0",
```

Comment:
This change is accidental or on purpose?

Reply:
I just ran `npm install`.